cookbook/form/form_customization.rst

```diff
@@ -114,6 +114,12 @@ fragment needed to render every part of a form:
   but the CSS classes applied are the ones used to display the forms horizontally
   (i.e. the label and the widget in the same row).
 
+.. caution::
+
+    When using the Bootstrap form themes and render the fields manually,
```

Comment:
I'm wondering if this should be "When using the Bootstrap form themes and rendering [...]".